lib/pages/search_page/search_page.dart

```diff
@@ -0,0 +1,257 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:sossoldi/providers/accounts_provider.dart';
+import 'package:sossoldi/providers/transactions_provider.dart';
+import '../../custom_widgets/transactions_list.dart';
+import '../../model/transaction.dart';
```

Comment:
Can you fit the first 2 imports paths to our code guidelines?
Just do the same as the last 2 imports, with the absolute path

Reply:
Sure